.travis.yml

```diff
@@ -22,10 +22,13 @@ matrix:
         - cargo test --all
     - language: node_js
       node_js: node
-      before_script: cd truffle
+      before_script:
+        - cd truffle
+        - npm install
```

Comment:
Can we use `yarn` and commit `yarn.lock` as well for faster and deterministic builds?